src/test/groovy/htmlpublisher/HtmlPublisherTest.groovy

```diff
@@ -13,7 +13,7 @@ public class HtmlPublisherTest extends HudsonTestCase {
      */
     public void testConfigRoundtrip() {
         def p = createFreeStyleProject();
-        def l = [new HtmlPublisherTarget("a", "b", "c", true, true, false), new HtmlPublisherTarget("", "", "", false, false, false)]
+        def l = [new HtmlPublisherTarget("a", "b", "c","", true, true, false), new HtmlPublisherTarget("", "", "","", false, false, false)]
```

Comment:
Could you add a space after the command before the quotes? It's missing a space in all the test files :)